docs/modules/indexes/retrievers/examples/weaviate-hybrid.ipynb

```diff
@@ -30,64 +39,127 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
    "id": "c10dd962",
    "metadata": {},
    "outputs": [],
    "source": [
     "import weaviate\n",
     "import os\n",
     "\n",
-    "WEAVIATE_URL = \"...\"\n",
+    "WEAVIATE_URL = os.getenv(\"WEAVIATE_URL\")\n",
     "client = weaviate.Client(\n",
     "    url=WEAVIATE_URL,\n",
-    ")"
+    "    auth_client_secret=weaviate.AuthApiKey(api_key=os.getenv(\"WEAVIATE_API_KEY\")),\n",
+    "    additional_headers={\n",
+    "        \"X-Openai-Api-Key\": os.getenv(\"OPENAI_API_KEY\"),\n",
+    "    },\n",
+    ")\n",
+    "\n",
+    "client.schema.delete_all()"
```

Comment:
Is it possible that a user would accidentally run this to delete their schema? Could we perhaps comment it out or put it in a markdown cell so that a user doesn't accidentally lose their data?

Reply:
I'll comment it out